direct-auth-samples/src/test/java/pages/RootPage.java

```diff
@@ -64,4 +67,24 @@ public boolean isLoginButtonDisplayed() {
     public boolean isRegistrationButtonDisplayed() {
         return registrationButton.isDisplayed();
     }
+
+    public boolean isLogoutButtonDisplayed() {
+        return logoutButton.isDisplayed();
+    }
+
+    public void waitForLoginButtonDisplayed(){
+        new WebDriverWait(driver, 5)
+                .until(ExpectedConditions.visibilityOf(logoutButton));
+
+    }
+
+    public boolean elementIsDisplayed(WebElement element){
+        try {
+        element.isDisplayed();
```

Comment:
Can `isDisplayed()` method return **false** without exception?..

Reply:
@VitaliiTytarenko-okta 
in this case element throws NoSuchElementException.

Reply:
From the JavaDoc:
```
Is this element displayed or not? This method avoids the problem of having to parse an element's "style" attribute.
Returns: Whether or not the element is displayed
```
Therefore, I believe that it **can** return `true/false` without exceptions.

Reply:
Another question: what will your method return if `isDisplayed()` returns false without any exceptions?

Reply:
Fixed